Dockerfile

```diff
@@ -2,10 +2,10 @@ FROM registry.access.redhat.com/ubi9/python-311:1-72.1724040033
 
 WORKDIR /app
 
-COPY --chown=1001:0 *.py requirements.txt ./
+COPY --chown=1001:0 . .
```

Comment:
nope, the Dockerfile should not contain docs, pngs, and other stuff from the repo